### DISCUSSION TITLE
SVG output not open by browsers

### DISCUSSION THREAD

**Discussion by bounaux** [explosion/spaCy#12327] [Help: Other Questions] [answered]
## How to reproduce the behaviour

follow sample code: 

 svg = displacy.render(doc, style='ent', jupyter=False)
output_path = Path("sentence.svg")
output_path.open("w", encoding="utf-8").write(svg)

## Your Environment
* Operating System: Windows 10
* Python Version Used: Python 3.10.1
* spaCy Version Used: 3.4.4
* Environment Information: python notebook

Result: the svg file is generated OK, but when try to open in browser this msg is displayed: 

This XML file does not appear to have any style information associated with it. The document tree is shown below.

This svg file does work with online SVG viewer: https://www.svgviewer.dev/

**Comment by svlandeg** [comment, marked as answer]
Hi! If you want to see the result in your browser, you can run this instead:
```
displacy.serve(doc, style='ent')
```
and it'll write the link to access on the console, for me that's http://localhost:5000/

**Reply by bounaux** [reply]
Ok cool thx! Seems it's just missing the HTML opening/closing tags I can add them before the write.
